examples/machining.yaml

```diff
@@ -0,0 +1,430 @@
+    '@type': ResourceSpecification
+    name: Unthreaded part 45
+    unit: unit:Number
+    substitutable: True
+
+  - '@id': machine:07819789-dd51-44c3-b35c-9210165bc832
+    '@type': ResourceSpecification
+    classifiedAs: https://www.wikidata.org/wiki/Q2211295 # drilling machine
+    name: Drill Press
+    unit: unit:Number
+    unitOfUse: unit:Hour # want a better way to do this!
+    substitutable: True
+
+  - '@id': machine:c404cff5-19c7-453d-b0d8-d8e96055bd0b
+    '@type': RecipeProcess
+    skos:note: Thread a part
+    time:hasDuration:
+      time:unitType: time:unitHour
+      time:numericDuration: 2
+    scalingFormula: D*M
+
+  - '@id': machine:c7897c39-7f05-4a5d-a487-80e130a2414b
+    '@type': RecipeFlow
+    action: produce 
+    resourceConformsTo: machine:a7e94dde-2997-451e-a7e1-8d364d5e6dee # threaded part
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 80 # minimum batch size to be worth it for setup
+    scalingFormula: Q*M
+    scalesPlan: True
+
+  - '@id': machine:13cf44e5-eaa1-4ad0-88d5-e2910a73f377
+    '@type': RecipeFlow
+    action: work 
+    resourceClassifiedAs: https://www.wikidata.org/wiki/Q192047 # machining
+    quantifiedAs:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 2
+    scalingFormula: Q*M
+
+  - '@id': machine:9f112621-3f13-4f48-a3e2-fd2ca99493cd
+    '@type': RecipeFlow
+    action: consume
+    resourceConformsTo: machine:2d350055-acba-4e17-92f8-8b126272ee40 # unthreaded part
+    quantifiedAs:
+      qudt:unit: unit:Number
+      qudt:numericValue: 80
+    scalingFormula: Q*M
+
+  - '@id': machine:ff208521-9fec-4bb4-a6d3-197f6735cb26
+    '@type': RecipeFlow
+    action: use
+    resourceConformsTo: machine:07819789-dd51-44c3-b35c-9210165bc832 # drill press
+    quantifiedAs:
+      qudt:unit: unit:Hour
+      qudt:numericValue: 2
+    scalingFormula: Q*M
+
+  # resources before
+
+  - '@id': machine:e1721a61-cd47-4556-84b9-8b1b81da15bf
+    '@type': EconomicResource
+    name: Drill Press 1
+    conformsTo: machine:07819789-dd51-44c3-b35c-9210165bc832
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': machine:e7482f32-25b5-4f79-934b-5d0db3a5b77d
+    '@type': EconomicResource
+    name: Drill Press 2
+    conformsTo: machine:07819789-dd51-44c3-b35c-9210165bc832
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  - '@id': machine:6fb358a3-2859-4d6a-a4fa-431603ee70f5
+    '@type': EconomicResource
+    name: Unthreaded part 45
+    conformsTo: machine:2d350055-acba-4e17-92f8-8b126272ee40
+    currentQuantity:
+      qudt:unit: unit:Number
+      qudt:numericValue: 1
+
+  # plan
+
+  # Multiplier = demand / recipe batch size, M = 1000 / 80 = 12.5
+  # first pass, create plan from recipe assuming no limits on time (this is the optimum because of the least setup time)
+  # second pass, capacity planning to modify plan to fit due date and person/machine availability, this is the plan
```

Comment:
Having two distinct scaling factors:
* number of machines and people
* duration of work and usage

Application could easily calculate 'on-the-fly' all the possible combinations, given specific number (1000 - or adjusted to full batches) it could even provide UI with a rectangle where one can drag a corner and ratio between factor changes while their products stays matching what it takes to produce that specific number.

If you create plan for 1 person and 1 machine from a recipe (duration 25h), how from there application supposed to help with modifying the plan to what you have below if the recipe seems to have the information needed for it.

Reply:
Yeah, there could be all kinds of apps to help create plans from recipes, and I've seen also a lot of tweaking and hands-on adjustment of plans. 

An app could do what I did, which was to take the straight line plan (25 hours), which gives you minimum machine setup time, then scrunch it to fit the deadline just like you are suggesting with the dragging the corner of the rectangle to get different ratios.  At the same time they would have to take into account other demands on machines, and the schedules of the people.  Since it is only working with classifications/specifications, not specific machines, it can take all available substitutable equipment and skills into account.

Reply:
> An app could do what I did, which was to take the straight line plan (25 hours), which gives you minimum machine setup time, then scrunch it to fit the deadline just like you are suggesting with the dragging the corner of the rectangle to get different ratios.

This sounds like a good candidate for prototyping exercise. I could do prototype which does it directly from Recipe and can scale volume of equipment and people as well as duration. You could give a shot to your approach and we could discuss our implementation experiences.
At this moment I think doing it directly from Recipe should come simpler and we don't get this intermediate plan which no one actually would ever want to execute in at any point in described scenario.

Reply:
If there's any way you could think about the core naming with me first, that would be very helpful.  I understand it might be different if you want many quantities and I want just one as the core.